rholang/src/main/scala/coop/rchain/rholang/interpreter/Registry.scala

```diff
@@ -165,12 +157,6 @@ class Registry(private val space: Runtime.RhoPureSpace,
     Seq(Quote(Par(exprs = Seq(EVar(FreeVar(0))), connectiveUsed = true))),
     freeCount = 1)
 
-  private val insertCallbackRef: Long = Runtime.BodyRefs.REG_INSERT_CALLBACK
-
-  private val deleteRef: Long             = Runtime.BodyRefs.REG_DELETE
-  private val deleteRootCallbackRef: Long = Runtime.BodyRefs.REG_DELETE_ROOT_CALLBACK
-  private val deleteCallbackRef: Long     = Runtime.BodyRefs.REG_DELETE_CALLBACK
-
```

Comment:
Those seem to be unrelated changes, please remove them from this commit - it's confusing as hell :)

Reply:
They are very much related and even necessary for everything to work. Because I moved where the `Registry` is being created (from the `dispatchTableCreator` to the `RholangAndScalaDispatcher.create` method) I couldn't use `testingDispatchTable` from the instance of `Registry`. I had to move it out from the instance to an object.

Reply:
I still don't see how that makes it part of fixing the effect type to Task. This could be one or two separate commits. It'd help with changing the wrong part of the commit (fix over Task) by having it separate.

Reply:
You are now arguing that my git hygiene is bad. I know it can improve, I am working on it.

Reply:
I'm very pleased with your git hygiene in general and I appreciate that you're improving even now. The point of my comment was to point out how you can improve further :) And provide rationale why it might be worthy (b/c it'd make your and our work easier; in this particular case - b/c I'd like us to change the 'wrong' part of this commit)